share/zm-rpcapi.lsb

```diff
@@ -27,7 +27,7 @@ STARMAN=`PATH="$PATH:/usr/local/bin" /usr/bin/which starman`
 . /lib/lsb/init-functions
 
 start () {
-    $STARMAN --user=$USER --group=$GROUP --error-log=$LOGFILE --pid=$PIDFILE --listen=$LISTENIP:$LISTENPORT --preload-app --daemonize $BINDIR/zonemaster_backend_rpcapi.psgi || exit 1
+    $STARMAN --daemonize --preload-app --user=$USER --group=$GROUP --pid=$PIDFILE --error-log=$LOGFILE --listen=$LISTENIP:$LISTENPORT $BINDIR/usr/local/bin/zonemaster_backend_rpcapi.psgi || exit 1
```

Comment:
I think this should be
`$BINDIR/zonemaster_backend_rpcapi.psgi`
and not
`$BINDIR/usr/local/bin/zonemaster_backend_rpcapi.psgi`